StreamNode/WebClient/src/routes/dashboard/index.js

```diff
@@ -139,27 +143,35 @@ class Dashboard extends Component {
                     <Title level={2} style={{marginBottom:0}}>MIXER</Title>
                 </Row>
 
-                <Row justify="center" align="middle">
-                    <Col span={this.state.isMobile ? 3 : 2} >
+                { //FILTRI
+                <Row justify= {this.state.isMobile ? "start" : "center"} align="middle">
+                    <Col span={this.state.isMobile ? 3 : 1} offset={this.state.isMobile && 1}>
                         <Row justify="center">
                         <Text strong >Filtro: </Text>
                         </Row>
                     </Col>
-                    <Col span={this.state.isMobile ? 18 : 20}>
-                        
+                    <Col span={this.state.isMobile ? 15 : 18}>
                         <Select defaultValue={deviceFilterNone} onChange={this.onChangeDevice.bind(this)} style={{width: "100%"}}>
-                            <Option value={deviceFilterNone}>None</Option>
+                            <Option value={deviceFilterNone}>All</Option>
                             {optionSourcesList}
                         </Select>
                     </Col>
+                    <Col offset={1} span={this.state.isMobile ? 2 : 2}>
+                        <Switch onChange={ () => this.setState({ isVertical: !this.state.isVertical }) } 
+                            checkedChildren="V"
+                            unCheckedChildren="H"
+                            defaultChecked={this.state.isVertical}
+                        />
+                    </Col>
                 </Row>
+                }
 
                 {   //VOLUMES DECK
                     this.state.VolumesLoaded 
                     
                     ?
 
-                    <Row justify={"start"}>
+                    <Row justify={"start"} style={{ margin: this.state.isVertical ? (!this.state.isMobile ? "0 5%" : "0 auto") : "0 auto"}}>
```

Comment:
Same responsive here, we should understand if is possible to do the same thing with Row component